module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,62 @@
+#include "AppendNewRecipe.hpp"
+
+#include <algorithm>
+#include <fstream>
+
+bool AppendNewRecipe(std::vector<std::string> steps,
```

Comment:
don;t copy vector

Reply:
I copied the function signature from the STL#4 lesson zadanie 1. I will correct it, however it should be also fixed on the website.